src/auth/cephx/CephxServiceHandler.cc

```diff
@@ -49,11 +51,12 @@ int CephxServiceHandler::start_session(const EntityName& name,
 
 int CephxServiceHandler::handle_request(
   bufferlist::const_iterator& indata,
+  size_t connection_secret_required_len,
   bufferlist *result_bl,
   uint64_t *global_id,
   AuthCapsInfo *caps,
   CryptoKey *psession_key,
-  CryptoKey *pconnection_secret)
+  std::string *pconnection_secret)
```

Comment:
Is there a reason for the weird order that separates the connection secret and its length?

Reply:
input vs output argument